src/utils/ProfileUtils.ts

```diff
@@ -125,6 +125,27 @@ export async function installModsToProfile(
 
 export async function parseYamlToExportFormat(yamlContent: string) {
     const parsedYaml = await yaml.parse(yamlContent);
+    if (!parsedYaml) {
+        throw new R2Error(
+            'Failed to parse yaml contents.',
+            'Yaml parsing failed when trying to import profile via file (The contents of export.r2x file are invalid).',
+            'Ensure that the profile import file isn\'t corrupted.'
+        )
+    }
+    if (!parsedYaml.profileName || typeof parsedYaml.profileName !== 'string') {
```

Comment:
Nitpick: `typeof parsedYaml.profileName !== 'string'` would suffice here since `undefined` is not a string.´?